source/manual/alerts/email-alert-api-unprocessed-content-changes.html.md

```diff
@@ -5,14 +5,12 @@ section: Icinga alerts
 subsection: Email alerts
 layout: manual_layout
 parent: "/manual.html"
-last_reviewed_on: 2020-07-16
+last_reviewed_on: 2020-08-05
 review_in: 6 months
 ---
 
-This alert is triggered when content changes are not processed within the time we would expect.
-
-* `warning` - `content_changes` unprocessed for over 90 minutes
-* `critical` - `content_changes` unprocessed for over 120 minutes
+This alert is triggered when content changes are not processed within 120 minutes. This
+may be fine and the emails may eventually go out, but it's worth investigating.
```

Comment:
I think we may mislead a bit by stating that this may be fine so soon in this description.

I'd suggest amending this a bit, based on the information that we'd probably want if we were a naïve person seeing this alert. I think I'd want something like:

```
This alert indicates that Email Alert API has [content changes that were not processed](link to worker) within 2 hours. Processing a content change is the means to establish which subscriber lists should be notified when GOV.UK content changes and the process to generate individual emails about the change in content. 

We generally expect a content change to be have processed within 30 minutes of it's creation. At 30 minute intervals a job, [RecoverLostJobsWorker], will try to requeue any content changes that have not been processed within an hour.

Thus, seeing this alert indicates that we likely have a problem that needs manual intervention. Potential causes would be:

- Particular or all Content Change failing to process due to some error
- Sidekiq not running or not operating correctly
- Content changes taking an abormally long time to process, which may indicate degraded system performance or an unexpectedly large task as part of processing. This scenario may resolve automatically.

To diagnose this problem, consult Sentry to see if there are errors reported.

If you find nothing conclusive in Sentry, go to Email Alert API sidekiq logs and check the jobs are running correctly.

If the problem persists, run the RecoverLostJobsWorker and/or ProcessContentChangeWorker directly to see if any problems occur.
```

Similar for messages